src/target/target_kind.cc

```diff
@@ -373,6 +373,8 @@ TVM_REGISTER_TARGET_KIND("hybrid", kDLCPU)  // line break
 
 TVM_REGISTER_TARGET_KIND("composite", kDLCPU).add_attr_option<Array<Target>>("devices");
 
+TVM_REGISTER_TARGET_KIND("ethos-u", kDLCPU).set_attr<Bool>("c_device_api", Bool(true));
```

Comment:
what about a target like Ethos-N, which could in theory be used from either C or C++ runtimes? should we just call this attribute `use_device_api`?

Reply:
I think my nervousness here stems from enabling the Device API when the runtime doesn't support it. Given the C++ runtime uses the `Module` nodes rather than C functions, `Target`s could enumerate support rather than enabling all Device APIs? 

I'm happy to change this to `use_device_api` to match the RFC as I think there's bigger problems with enumerating the internal support for runtimes and executors but curious on your thoughts?

Reply:
aha so :) we are going to need to find a way to encode Device API calls in TIR which is agnostic to the runtime, i think. There is a way to get the `Module` instance in C++ runtime given a TVMDevice.